### PR TITLE
Paramedic Hardsuit 2: Monstermos Compensation Edition

### PR DESCRIPTION
**Pull request by Nickvr628**
See #8504 for all discussion and debate, this one is merge conflict free and merge ready.

Recreated at the request of Smudgels and others. Monstermos means the station lacks air a lot more than normal, and paramedics cant rescue people if they die from a vacuum. Hence the need for their spacesuit. The design is based on the Cyberpunk 2077 Trauma Team suits, who are cool future paramedics.

A new Paramedic Hardsuit is available to paramedics. Designed for when paramedics have to go to a more dangerous than normal location to save people, like space, the front line of a blob fight, or Reebe. Has slightly better armor than the normal medical hardsuit, but does not have virus or rad protection. 
 
image
Credit to Partheo for the awesome new sprite 

:cl:  
rscadd: A hardsuit for Paramedics. Good for rescuing people from dangerous situations. Credit to Partheo for the sprites.
/:cl:

### CODE REVIEW

**Comment by Partheo** [comment]
Looks a bit washed out overall. Helmet could be a bit more helmet-lookin, as it seems kinda flat.

Not sure about the red parts?

Design is cool overall, and the colors are good too, but I feel there needs to be more definition.

**Comment by Featherfield** [comment]
Spawning with maint and having a free spacesuit sounds like powergamers wet dream job...
just ask for EVA from hop or such

**Comment by Nickvr628** [comment]
You mean like all the other jobs that spawn with maint access and an available hardsuit?

Powergaming gets you bwoinked and/or banned. Until we get changes to monstermos, paramedics getting some way to deal with the vacuum of space is a necessity.

**Comment by Hopekz** [comment]
The same argument I had in your previous PR before still stands just in-case you're curious.

TL;DR:
- Make the hardsuit a spacesuit instead with no armor values and no thick armor.

would be nice but not as important as the first one:
- make it have 2 pieces instead of 1 so it isn't more desirable vs a 2 piece suit otherwise this becomes a high-value item to steal for others vs EVA suits. (Also stealing from EVA is more dangerous than stealing from paramedics)

**Comment by Nickvr628** [comment]
image

Thank you Partheo for the new sprites!

**Comment by Smudged** [comment]
> Spawning with maint and having a free spacesuit sounds like powergamers wet dream job...
> just ask for EVA from hop or such

I do think we should trust paramedics with at least some responsability. 
Other servers give paramedics suits we're seemingly behind on the curb

**Comment by Idinuum** [comment]
after some consideration, not really bothered by the combat focused armor values.

however, would really prefer if this was a space suit and not a hard suit

**Comment by Nickvr628** [comment]
Spacesuits are civilian in nature and nonspecific. All the specialized or departmental-specific suits are hardsuits. There isn’t really a lore or IC reason it shouldn’t be called a hardsuit over a spacesuit.

**Comment by zxmongoose** [comment]
I think it'd be ready good if paramedics got some spaceworthy gear, I actually agree with Hopek that it probably shouldn't have any armor value or a vary small one(like a bulkier, loose-fitting space suit, which fits the sprite I think, or it could be slimmer and more in line with other actual hardsuits, but made without reinforcing materials and such)

**Comment by Nickvr628** [comment]
Again, the armor values are slightly more than EVA Suits and medical hardsuits. They are well below actual combat hardsuits like security’s, and are pretty close the engineering hardsuit.

**Comment by Xantam-13** [comment]
Against this, there are space suits in EVA which the AI can let you get to protect human life if you explain you're there to recover a body. 9/10 times - paramedics do not go to space to recover bodies anyway unless it's Omegastation, and even that is because it's embedded into an asteroid with loot. 

The only way I'd be for this is if the suits deteriorate after X amount of time/uses, making them worthless after 10 minutes or something. Gives the paramedic an actual time-frame to get the suit on, in space, body and back to the station.

**Comment by Nickvr628** [comment]
Try playing paramedic during any destructive antag round, anyone that dies from the station having no air is essentially impossible to save. Other servers give paramedics the ability to rescue people in space, and I am not making a snowflake "worse hardsuit" that is gimped because people are afraid of powergamers stealing them.

Admins bwoink engineers fucking off with hardsuits when they are needed on station, paramedics will be no different.

**Comment by MacHac** [comment]
Paramedics are obnoxious to antags anyway because they surprise you after every kill to steal the body.  Letting them stalk you even in space may be going too far.  Besides, the fact that you even thought to elevate the armor values above the default hints at the true purpose of this PR: letting paramedics validhunt in the one place they currently can't.

Paramedics - You don't have to be a validhunter, but it helps.™

_Disclaimer: only partially serious.  If this doesn't apply to you, great.  It applies to someone and I'm not looking forward to meeting them in game._

**Comment by Smudged** [comment]
> Against this, there are space suits in EVA which the AI can let you get to protect human life if you explain you're there to recover a body. 9/10 times - paramedics do not go to space to recover bodies anyway unless it's Omegastation, and even that is because it's embedded into an asteroid with loot.
> 
> The only way I'd be for this is if the suits deteriorate after X amount of time/uses, making them worthless after 10 minutes or something. Gives the paramedic an actual time-frame to get the suit on, in space, body and back to the station.

I think this doesn't touch on the fact of how prevalent monstermos is now, and how other stations do give paramedics the responsability of having space suits. 
The EVA hardsuits can just become a hassle since every single round, you as a paramedic have to ask for what is an essential piece of gear to do your job now.

**Comment by Smudged** [comment]
> Paramedics are obnoxious to antags anyway because they surprise you after every kill to steal the body. Letting them stalk you even in space may be going too far. Besides, the fact that you even thought to elevate the armor values above the default hints at the true purpose of this PR: letting paramedics validhunt in the one place they currently can't.
> 
> Paramedics - You don't have to be a validhunter, but it helps.™
> 
> _Disclaimer: only partially serious. If this doesn't apply to you, great. It applies to someone and I'm not looking forward to meeting them in game._

If paramedics decide to validhunt, ban them? 

Your argument goes into that there's a small population of paramedics that valid hunt, so therefore, normal paramedics can't get this feature. 
I think that's just ridiculous that a small population of player can just ruin it for every other rule-abiding player.

**Comment by Smudged** [comment]
> Again, the armor values are slightly more than EVA Suits and medical hardsuits. They are well below actual combat hardsuits like security’s, and are pretty close the engineering hardsuit.

I'm hesitant to make it better than the CMO's hardsuit, if you could make it inbetween the CMO's hardsuit, and the eva suit, that miight be better.

**Comment by Nickvr628** [comment]
Why? There are two paramedic slots, is only one allowed to do their job?

**Comment by SomeguyManperson** [comment]
did anyone even check the icondiffbot holy fuck
